pkg/couchdb/ui/src/@types/ambient.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '*.svelte' {
+	export { SvelteComponentDev as default } from 'svelte/internal';
+}
```

Comment:
Delete (you can delete the entire `@types` folder, we can add it later if needed)